python/lsst/ts/xml/enums/MTRotator.py

```diff
@@ -71,11 +76,24 @@ class EnabledSubstate(enum.IntEnum):
     MOVING_POINT_TO_POINT = enum.auto()
     SLEWING_OR_TRACKING = enum.auto()
     CONTROLLED_STOPPING = enum.auto()
-    INITIALIZING = enum.auto()
-    RELATIVE = enum.auto()
+    INITIALIZING = enum.auto()  # Deprecated in rotator
+    RELATIVE = enum.auto()  # Deprecated in rotator
     CONSTANT_VELOCITY = enum.auto()
 
 
+class FaultSubstate(enum.IntEnum):
+    """Controller substate for the FAULT state.
+
+    Value reported in ``telemetry.fault_substate``.
+
+    This is enum ``FaultSubStates`` in Simulink model.
+    """
+
+    EMERGENCY_STOPPING = 0
```

Comment:
Are you sure you want `EMERGENCY_STOPPING` to be state with value 0? 

I think I would expect something like `UNKNOWN` or maybe ever `NO_ERROR` as the lowest enum value.

Reply:
This is to be consistent with the `FaultSubStates` added by Julen:
https://github.com/lsst-ts/ts_mt_hexRot_simulink/blob/develop/commonScripts/FaultSubStates.m

I think it might be better  to keep the same.